.travis.yml

```diff
@@ -67,6 +67,11 @@ install:
   - export CGO_LDFLAGS="$CGO_LDFLAGS -L`pwd` -lrocksdb"
   - export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:`pwd`"
 
+before_script:
+  - export CHERAMI_PACKAGES=`go list ./... | grep -ve vendor | grep -ve test | sed "s#github.com/uber/cherami-server#.#"` 
```

Comment:
agree with @venkat1109 move this to a "make lint" or something within the makefile and you can use "ALL_SRC" there to get all the files..